septima-js-data/src/main/java/com/septima/entities/SqlEntity.java

```diff
@@ -174,16 +176,20 @@ public String getName() {
     }
 
     public SqlQuery toQuery() {
-        Objects.requireNonNull(sqlText, "Sql query text missing.");
-        if (sqlText.isEmpty()) {
-            throw new IllegalStateException("Empty Sql query text is not supported");
-        }
-        if (customSqlText != null && !customSqlText.isEmpty()) {
-            Logger.getLogger(SqlEntity.class.getName()).log(Level.INFO, "Entity sql was substituted with customSql while compiling entity {0}", name);
+        String sqlClause;
+        if (customSqlText != null && !customSqlText.isBlank()) {
```

Comment:
customSqlText != null -> Objects.nonNull(customSqlText)

Reply:
What for?